README.md

```diff
@@ -11,14 +11,49 @@ The Dapr CLI allows you to setup Dapr on your local dev machine or on a Kubernet
 
 ### Prerequisites
 
-On default, during initialization the Dapr CLI will install the Dapr binaries as well as setup a developer environment to help you get started easily with Dapr. This environment uses Docker containers, therefore Docker needs to be installed. If you prefer to run Dapr without this environment and no dependency on Docker, after installation of the CLI make sure to follow the instructions to initialize Dapr using [slim init](#slim-init).
+On default, during initialization the Dapr CLI will install the Dapr binaries as well as setup a developer environment to help you get started easily with Dapr. This environment uses `Docker`, `Podman` or `Containerd` containers, therefore container runtime needs to be installed. If you prefer to run Dapr without this environment and without any container dependencies, after installation of the CLI make sure to follow the instructions to initialize Dapr using [slim init](#slim-init).
 
-Note, if you are a new user, it is strongly recommended to install Docker and use the regular init command.
+#### Note, if you are a new user, it is strongly recommended to install Docker and use the regular init command.
 
 * Install [Docker](https://docs.docker.com/install/)
 
 >__Note: On Windows, Docker must be running in Linux Containers mode__
 
+#### If you want to use podman as a runtime, then please refer to the installation.
+
+* Install [Podman](https://podman-desktop.io/docs/Installation)
+
+#### If you want to use containerd as a runtime, then please refer to the installation.
```

Comment:
This can be simpler.

#### Choosing a container runtime
##### Docker (recommended for new users)
Docker instructions
##### Podman
Podman instructions
##### Containerd
Containerd instructions

Reply:
Okay, thanks very much!